msteams-platform/bots/how-to/format-your-bot-messages.md

```diff
@@ -10,6 +10,9 @@ ms.author: anclear
 
 Message formatting enables you to bring out the best in bot messages. You can format your bot messages to include rich cards that are attachments that contain interactive elements, such as buttons, text, images, audio, video, and so on.
 
+> [!NOTE]
+> The bot message size limit is 50 KB. You can't send bot message if its size exceeds 50 KB.
```

Comment:
This is missing the important info of what counts towards this limit. For example, do data URI encoded images contribute to this limit? Whitespace?
Also, 50KB in what encoding?

Reply:
Limit includes the entire message payload encoded as UTF-16